day1/scripts/agent.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
+yum install vim -y;
+yum install http://repo.zabbix.com/zabbix/3.2/rhel/7/x86_64/zabbix-release-3.2-1.el7.noarch.rpm -y;
+yum install zabbix-agent -y;
+yum install zabbix-sender -y;
+systemctl start zabbix-agent;
```

Comment:
Sorry it was my mistake, I make agent.sh without sed at first and then only add this block. Of course I should start service in the end.